src/components/TouchableOpacity.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * https://github.com/facebook/react-native/blob/master/Libraries/Components/Touchable/TouchableWithoutFeedback.js
+ */
+import React from 'react';
+import EdgeInsetsPropType from '../ropTypes/EdgeInsetsPropType';
+import View from './View';
+// import Touchable from './Touchable';
```

Comment:
Sure, I will remove it.
It existed in the 'TouchableWithoutFeedback' so I kept it. Same with the mixins line. Should I keep that or remove it too?